app/src/main/java/com/brainwallet/ui/screens/welcome/WelcomeViewModel.kt

```diff
@@ -108,6 +109,13 @@ class WelcomeViewModel(
                     )
                 }
             }
+
+            is WelcomeEvent.OnLoad -> {
+                /**
+                 * generate temp random seed phrase
+                 */
+                BRWalletManager.getInstance().generateRandomSeed(event.context, true)
```

Comment:
generate random seed phrase and save it to temp (second parameter is boolean `isTemp`)